### PR TITLE
First step towards fixing #272

### PR DESCRIPTION
**Pull request by TheDauntless**
Fixed the consistency part for #272

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit 2cf0b109f22aa0c476a8b565db0df49066516ec6: Broken link result: 0 .
  
 Markdown result errorlength: 9 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by commjoen** [comment]
Can you please fix the markdown issues?

File: Document-zhtw/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 1]
```diff
@@ -30,9 +30,9 @@
...
-- OWASP Thread modelling: <https://www.owasp.org/index.php/Application_Threat_Modeling>
-- OWASP Secure SDLC Cheat Sheet: <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
-- Microsoft SDL: <https://www.microsoft.com/en-us/sdl/>
-- NIST SP 800-57: <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
+- OWASP Mobile Top 10: M10 (Extraneous Functionality) - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M10-Extraneous_Functionality>
+- OWASP Security Architecture cheat sheet - <https://www.owasp.org/index.php/Application_Security_Architecture_Cheat_Sheet>
+- OWASP Thread modelling - <https://www.owasp.org/index.php/Application_Threat_Modeling>
+- OWASP Secure SDLC Cheat Sheet - <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
+- Microsoft SDL - <https://www.microsoft.com/en-us/sdl/>
+- NIST SP 800-57 - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```
Comment (cpholguera) [inline Document-zhtw/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]:
```suggestion
- NIST SP 800-57 (Recommendation for Key Management) - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```

File: Document-ja/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ OWASP モバイルセキュリティテストガイドでは、このセクシ
 
 詳しくは以下の情報を参照してください。
 
-- OWASP Mobile Top 10: M8 - Code Tampering: <https://www.owasp.org/index.php/Mobile_Top_10_2016-M8-Code_Tampering>
-- OWASP Mobile Top 10: M9 - Reverse Engineering: <https://www.owasp.org/index.php/Mobile_Top_10_2016-M9-Reverse_Engineering>
-- OWASP Reverse Engineering Threats - <https://www.owasp.org/index.php/Technical_Risks_of_Reverse_Engineering_and_Unauthorized_Code_Modification>
+- OWASP Mobile Top 10: M8 (Code Tampering) - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M8-Code_Tampering>
+- OWASP Mobile Top 10: M9 (Reverse Engineering) - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M9-Reverse_Engineering>
+- WASP Reverse Engineering Threats - <https://www.owasp.org/index.php/Technical_Risks_of_Reverse_Engineering_and_Unauthorized_Code_Modification>
```
Comment (cpholguera) [inline Document-ja/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md]:
```suggestion
- OWASP Reverse Engineering Threats - <https://www.owasp.org/index.php/Technical_Risks_of_Reverse_Engineering_and_Unauthorized_Code_Modification>
```

File: Document-ja/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 3]
```diff
@@ -30,9 +30,9 @@ MASVS-L1 および MASVS-L2 の要件を以下に記します。
...
-- OWASP Thread modelling: <https://www.owasp.org/index.php/Application_Threat_Modeling>
-- OWASP Secure SDLC Cheat Sheet: <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
-- Microsoft SDL: <https://www.microsoft.com/en-us/sdl/>
-- NIST SP 800-57: <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
+- OWASP Mobile Top 10: M10 (Extraneous Functionality) - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M10-Extraneous_Functionality>
+- OWASP Security Architecture cheat sheet - <https://www.owasp.org/index.php/Application_Security_Architecture_Cheat_Sheet>
+- OWASP Thread modelling - <https://www.owasp.org/index.php/Application_Threat_Modeling>
+- OWASP Secure SDLC Cheat Sheet - <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
+- Microsoft SDL - <https://www.microsoft.com/en-us/sdl/>
+- NIST SP 800-57 - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```
Comment (cpholguera) [inline Document-ja/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]:
```suggestion
- NIST SP 800-57 (Recommendation for Key Management) - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```

File: Document-fr/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 4]
```diff
@@ -30,9 +30,9 @@ Les exigences pour MASVS-L1 et MASVS-L2 sont listées ci-dessous.
...
-- OWASP Thread modelling: <https://www.owasp.org/index.php/Application_Threat_Modeling>
-- OWASP Secure SDLC Cheat Sheet: <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
-- Microsoft SDL: <https://www.microsoft.com/en-us/sdl/>
-- NIST SP 800-57: <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
+- OWASP Mobile Top 10: M10 - Fonctions superflues - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M10-Extraneous_Functionality>
+- OWASP Security Architecture cheat sheet - <https://www.owasp.org/index.php/Application_Security_Architecture_Cheat_Sheet>
+- OWASP Thread modelling - <https://www.owasp.org/index.php/Application_Threat_Modeling>
+- OWASP Secure SDLC Cheat Sheet - <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
+- Microsoft SDL - <https://www.microsoft.com/en-us/sdl/>
+- NIST SP 800-57 - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```
Comment (cpholguera) [inline Document-fr/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]:
```suggestion
- NIST SP 800-57 (Recommendation for Key Management) - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```

File: Document-es/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 5]
```diff
@@ -30,9 +30,9 @@ A continuación, se enumeran los requerimientos para MASVS-L1 y MASVS-L2.
...
-- Modelado de Amenazas de OWASP: <https://www.owasp.org/index.php/Application_Threat_Modeling>
-- "Cheat sheet" para el ciclo de desarrollo seguro de OWASP: <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
-- Microsoft SDL: <https://www.microsoft.com/en-us/sdl/>
-- NIST SP 800-57: <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
+- OWASP Top 10 Móvil: M10 (Funcionalidades Extrañas) - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M10-Extraneous_Functionality>
+- "Cheat sheet" de la Seguridad en la Arquitectura de OWASP - <https://www.owasp.org/index.php/Application_Security_Architecture_Cheat_Sheet>
+- Modelado de Amenazas de OWASP - <https://www.owasp.org/index.php/Application_Threat_Modeling>
+- "Cheat sheet" para el ciclo de desarrollo seguro de OWASP - <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
+- Microsoft SDL - <https://www.microsoft.com/en-us/sdl/>
+- NIST SP 800-57 - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```
Comment (cpholguera) [inline Document-es/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]:
```suggestion
- NIST SP 800-57 (Recommendation for Key Management) - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```

File: Document-de/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 6]
```diff
@@ -33,9 +33,9 @@ Die Anforderungen für MASVS-L1 und MASVS-L2 sind nachfolgend aufgelistet.
...
-- OWASP Thread modelling: <https://www.owasp.org/index.php/Application_Threat_Modeling>
-- OWASP Secure SDLC Cheat Sheet: <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
-- Microsoft SDL: <https://www.microsoft.com/en-us/sdl/>
-- NIST SP 800-57: <https://csrc.nist.gov/publications/detail/sp/800-57-part-1/rev-4/final>
+- OWASP Mobile Top 10: M10 (Extraneous Functionality) - <https://www.owasp.org/index.php/Mobile_Top_10_2016-M10-Extraneous_Functionality>
+- OWASP Security Architecture cheat sheet - <https://www.owasp.org/index.php/Application_Security_Architecture_Cheat_Sheet>
+- OWASP Thread modelling - <https://www.owasp.org/index.php/Application_Threat_Modeling>
+- OWASP Secure SDLC Cheat Sheet - <https://www.owasp.org/index.php/Secure_SDLC_Cheat_Sheet>
+- Microsoft SDL - <https://www.microsoft.com/en-us/sdl/>
+- NIST SP 800-57 - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```
Comment (cpholguera) [inline Document-de/0x06-V1-Architecture_design_and_threat_modelling_requireme.md]:
```suggestion
- NIST SP 800-57 (Recommendation for Key Management) - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
```

**Comment by OMTGreleaser** [comment]
Results for commit 1885467495595745409486d150e25c8fd1b7f634: Broken link result: 0 .
  
 Markdown result errorlength: 9 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**